task.dart

```diff
@@ -0,0 +1,116 @@
+import 'dart:io';
+
+class Task {
+  int id;
+  String? title;
+  String? description;
+  bool isCompleted;
+
+  Task(this.id, this.title, this.description, this.isCompleted);
+}
+
+class TaskManager {
+  List<Task> tasks = [];
+  int id = 1;
+
+  //Add tasks
+  void addTask(String? title, String? description, bool isCompleted) {
+    tasks.add(Task(id, title, description,
+        isCompleted)); //takes constructor/object as a input
+
+    id++;
```

Comment:
since you are accepting null values, validate before adding
```suggestion
    if(title == null || description == null || isCompleted == null)
    {
        print("Not posssible")
        return;
    }
    tasks.add(Task(id, title, description,
        isCompleted)); //takes constructor/object as a input

    id++;
```